spec/facility_spec.rb

```diff
@@ -2,25 +2,284 @@
+
+      expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+      
+      expected = {
+                :written=>false,
+                :license=>false, 
+                :renewed=>false}
+      expect(@registrant_1.license_data).to eq(expected)
+      
+      @facility_1.add_service("Written Test")
+
+      expect(@facility_1.administer_written_test(@registrant_1)).to eq(true)
+      
+      expected = {
+              :written=>true,
+              :license=>false, 
+              :renewed=>false
+            }
+
+      expect(@registrant_1.license_data).to eq(expected)
+      expect(@registrant_2.age).to eq(16)
+      expect(@registrant_2.permit?).to eq(false)
+      expect(@facility_1.administer_written_test(@registrant_2)).to eq(false)
+
+      @registrant_2.earn_permit
+
+      expect(@facility_1.administer_written_test(@registrant_2)).to eq(true)
+      expected = {
+        :written=>true,
+        :license=>false, 
+        :renewed=>false}
+      expect(@registrant_2.license_data).to eq(expected)
+      
+      expect(@registrant_3.age).to eq(15)
+      expect(@registrant_3.permit?).to eq(false)
+      expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+
+      @registrant_3.earn_permit
+
+      expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+      expected = {
+        :written=>false,
+        :license=>false, 
+        :renewed=>false
+      }
+      expect(@registrant_3.license_data).to eq(expected)
+    end
+  end
+
+  describe "administer road test" do
+    it "can administer road tests" do
+
+      expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+
+      @registrant_3.earn_permit
+
+      expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+      
+      expected_3 = {
+                :written=>false, 
+                :license=>false, 
+                :renewed=>false
+              }
+      expect(@registrant_3.license_data).to eq(expected_3)
+
+      expect(@facility_1.administer_road_test(@registrant_1)).to eq(false)
+      
+      @facility_1.add_service("Written Test")
+      @facility_1.add_service("Road Test")
+
+      expect(@facility_1.services).to eq(["Written Test", "Road Test"])
+
+      @facility_1.administer_written_test(@registrant_1)
+      
+      expect(@facility_1.administer_road_test(@registrant_1)).to eq(true)
+
+      expected_1 = {
+                written: true, 
+                license: true, 
+                renewed: false
+      }
+      expect(@registrant_1.license_data).to eq(expected_1)
+     
+      @registrant_2.earn_permit
+      @facility_1.administer_written_test(@registrant_2)
+
+      expect(@facility_1.administer_road_test(@registrant_2)).to eq(true)
+
+      expected_2 = {
+         written: true, 
+         license: true,
+         renewed: false
+      }
+
+      expect(@registrant_2.license_data).to eq(expected_2)
+
+    end
+  end
+
+              # renew_license
```

Comment:
I would have your annotations flush, meaning don't have this line (229) tabbed in.

Reply:
Can you explain the reasoning behind that? To me it makes sense for easier readability.

Reply:
We don't want to add tabs or white space where we don't need it. If you are annotating it should just be a brief statement of what you are wanting to do in alignment with the code block that it is referencing.